elasticsearch/src/main/fabric8/deployment.yml

```diff
@@ -0,0 +1,27 @@
+spec:
+  replicas: ${replicas}
+  template:
+    spec:
+      containers:
+      - name: elasticsearch
+        image: ${fabric8.dockerUser}elasticsearch-k8s:${elasticsearch.version}
+        env:
+          - name: SERVICE
+            value: elasticsearch-masters
+        volumeMounts:
+          - name: data
+            mountPath: /usr/share/elasticsearch/data
+        livenessProbe:
+          tcpSocket:
+            port: 9300
+          initialDelaySeconds: 30
+        readinessProbe:
+          httpGet:
+            port: 9200
+            path: /_template/logstash
+          initialDelaySeconds: 5
+      - name: logstash-template
+        image: ${fabric8.dockerUser}elasticsearch-logstash-template:${project.version}
```

Comment:
we also need to be able to swizzle when staging multiple releases to use a staging registry versus the real registry (e.g. testing / staging / production registries). Docker registries are unfortunately the one thing we can't use kubernetes services for :(

I did some code to swizzle registries in docker images in kubernetes resources somewhere; though can't remember where ;) @rawlingsj can you remember by any chance? Struggling to remember where that code went ;)